lib/src/summarize.dart

```diff
@@ -4,23 +4,235 @@
 class Summarizer {
   _SummarizeToken storage;
-
+  String _code;
+  String _css;
+  String _html;
   bool resultsPresent;
-
-  Summarizer(String input, [AnalysisResults analysis]) {
-    if (input == null) throw new ArgumentError("Input can't be null.");
+  int randomizer;
+  
+  static Map<String, List<int>> cuttoffs = {
+    'size': [8, 30, 1000], //0-7 = small, 8 - 29 = decent, 29+ = gigantic
+    'errorCount': [1,5,100]
+  };
+  
+  static Map<String, String> codeKeyWords = {
+    'await' : 'await',
+    'async' : 'async',
+    'rpc' : 'RESTful serverside app'
+  };
+  
+  static Map<String, String> additionKeyWords = {
+      'pirate' : 'pirates',
+      'bird' : 'birds',
+      'llama' : 'llamas',
+  };
+    
+  static Map<String, List<String>> categories = {
+    /// This [size] [codeQuantifier] contains [error] errors and warnings.
+    'size-2': ['gigantic', 'Jupiterian sized', 'immense', 
+      'massive', 'enormous', 'huge', 'epic', 'humongous'],
+    'size-1': ['decently sized', 'exceptional', 'awesome', 
+      'amazing', 'visionary', 'legendary'],
+    'size-0': ['itty-bitty', 'miniature', 'tiny', 'pint-sized'],
+    'compiledQuantifier': ['DART program', 'pad'],
+    'failedQuantifier': ['assemblage of characters', 'series of strings', 'grouping of letters'],
+    'errorCount-2':['many', 'a motherload of', 'copious amounts of', 'unholy quantities of'],
+    'errorCount-1':['some', 'a few', 'sparse amounts of', 'very few instances of'],
+    'errorCount-0':['zero', 'no', 'a nonexistent amount of', '0.0000', '0'],
+    'use' : ['demonstrates', 'contains', 'illustrates', 'depicts'],
+    'code-0': ['it'],
+    'code-1': ['It'],
+  };
+  
+  Summarizer(String input, {String html, String css, AnalysisResults analysis}) {
+    if (input == null) throw new ArgumentError('Input cannot be null.');
     resultsPresent = !(analysis == null);
+    _code = input;
+    _html = html;
+    _css = css;
+    MD5 encryptor = new MD5();
+    encryptor.add(input.codeUnits);
+    randomizer = _sumList(encryptor.close());
     storage = new _SummarizeToken(input, analysis);
   }
+  
+  int _sumList(List<int> list) {
+    int sum = 0;
+    for (int number in list) {
+      sum += number;
+    }
+    return sum;
+  }
+  
+  String _categorySelector(String category, int itemCount) {
+    if (category == 'size' || category == 'errorCount') {
+      List<int> maxField = cuttoffs[category];
+      int counter = 0;
+      while(counter < maxField.length) {
+        if (itemCount < maxField[counter]) {
+          return '${category}-${counter}';
+        }
+        counter++;
+      }
+      return '${category}-${counter - 1}';
+    }
+    else {
+      return null;
+    }
+  }
+  
+  String _wordSelector(String category) {
+    if (categories.containsKey(category)) {
+      List<String> returnSet = categories[category];
+      return returnSet.elementAt(randomizer % returnSet.length);
+    }
+    else {
```

Comment:
This `else` should be on the same line as the `}`. You might want to run this file through the formatter - `dartfmt -w lib/src/summarize.dart`.